src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/GraphSymbolUtils.cs

```diff
@@ -557,7 +557,8 @@ public static bool IsGraphExtensionBaseType(this ITypeSymbol type)
 		private static bool IsGraphWithPrimaryDacBaseGenericType(INamedTypeSymbol type) =>
 			type.TypeArguments.Length >= 2 && type.Name == TypeNames.PXGraph;
 
-        public static Tuple<MethodDeclarationSyntax,IMethodSymbol> GetGraphExtensionInitialization(this INamedTypeSymbol typeSymbol, PXContext pxContext, CancellationToken cancellation = default)
+        public static (MethodDeclarationSyntax node, IMethodSymbol symbol) GetGraphExtensionInitialization
```

Comment:
Properties in `ValueTuple` should be declared in CamelCase.
Sorry for the late mention, we forgot to include it into the code style.

Reply:
Ok